PeerioTechnologies/peerio-mobile
# Feat signout untrust

### Linked PRs in other repos
https://github.com/PeerioTechnologies/peerio-icebear/pull/91
### Relevant issue tracker links
https://app.clubhouse.io/peerio/story/358/improve-signout-to-support-latest-sdk-and-untrust
### Manual testing instructions
Verify that signout still works correctly.
Verify that parameter passed to SDK through `User.current.signout(popupResult.checked);` sends the correct value of the checkbox (true for checked, false for unchecked).
### Breaking changes
N / A
### Additional notes
